src/dynamic-handle.ts

```diff
@@ -51,12 +52,25 @@ export const createDynamicHandler = (app: AnyElysia) => {
 				request,
 				path,
 				qi,
-				redirect
+				redirect,
+				error,
 			}
 		) as unknown as Context & {
 			response: unknown
 		}
 
+		const getLocalError = (response: unknown, hookError: HookContainer[]): Result | undefined => {
+			if (response instanceof ElysiaCustomStatusResponse) {
```

Comment:
Can I be sure that if `response` is an `ElysiaCustomStatusResponse` then it's surely an error?

Reply:
yes, `ElysiaCustomStatusResponse` is only return from `Elysia.error`